usr/sbin/so-netsniff-ng-cron

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Define initial variables
+SENSORTAB="/etc/nsm/sensortab"
+INTERFACES=$(grep -v '#' $SENSORTAB | awk '{print $1}')
+NETSNIFF_ENABLED="no"
+NETSNIF_RUNNING="no"
+OS_DATE=$(date +"%Y-%m-%d")
+
+# Check to see if netsniff-ng is enabled.
+for i in $INTERFACES; do
+	if grep -q 'PCAP_ENABLED="yes"' /etc/nsm/$i/sensor.conf; then
+		NETSNIFF_ENABLED="yes"
+	fi
+done
+
```

Comment:
I think all the code below should go inside the for-loop so that you're operating on each interface independently.  (Suppose a machine has multiple sniffing interfaces enabled and the first one has `PCAP_ENABLED="yes"` but the second one has `PCAP_ENABLED="no"`.)